packages/wells/src/client/model/Wellbore.ts

```diff
@@ -48,12 +48,6 @@ export interface Wellbore {
    * @memberof Wellbore
    */
   id: number;
-  /**
-   * Wellbore description
-   * @type {string}
-   * @memberof Wellbore
-   */
-  description?: string;
```

Comment:
Didn't someone add the description for wellbores last week on request from discover?